intel-retail/automated-vending
# feat(ms-authentication): Update ms-authentication service to EdgeX v3.0

## PR Checklist


- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below) N/A

## What are you changing?


## Issue this PR will close

close: #**124**

closes #124 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

 - `cd ms-authentication` and run `make docker` to build the ms-authentication microservice
 - Optional: comment out lines 32 to End of File of `docker-compose.ac.yml` (Other services haven't been updated yet and will fail)
- Run `make run`, the ms-authentication service should spin up
- The service startup "This microservice checks if ID numbers from REST requests are authenticated" will be shown in the container logs

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-vending )
